backend/azureblob/azureblob.go

```diff
@@ -1689,8 +1700,25 @@ func (o *Object) Update(ctx context.Context, in io.Reader, src fs.ObjectInfo, op
 		}
 	}
 
+	// calculate size of parts/blocks
+	partSize := int(o.fs.opt.ChunkSize)
+
+	uploadParts := o.fs.opt.MaxUploadParts
+        if uploadParts < 1 {
+                uploadParts = 1
+        } else if uploadParts > maxUploadParts {
```

Comment:
We've made max_upload_parts a config item, but here we override it with `maxUploadParts` we have as a constant.

We should probably either not have the config item, or don't override its maximum value, since the point of having a config value is for microsoft to say - the limit is now 100,000 and people can update their configs to match.

Thoughts?

Reply:
I lean towards that we should not introduce tuning of what has been provided as fixed scaling number.
Proceed by updating this branch to just to use the constant?

Reply:
Remove the configurable sounds fine to me - that would be in line with the S3 backend for example.